osdk/src/utils.rs

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use std::ffi::OsStr;
+use std::path::Path;
+use std::process::Command;
+
+use crate::error::Errno;
+use crate::error_msg;
+
+// TODO: switch to github url
+pub const ASTER_FRAME_DEP: &str =
+    "aster-frame = { git = \"https://github.com/asterinas/aster-for-osdk\", rev = \"89397bd\" }";
```

Comment:
Now that this crate is moved into the Asterinas repository, the crate don't have to download the source code of Asterinas.

Reply:
This `ASTER_FRAME_DEP` only affects the `new` command now. When creating a new crate with `cargo osdk new`, the `ASTER_FRAME_DEP` will be added to the `Cargo.toml` file of the new crate. Since the new crate may exist outside of Asterinas, it can only depend on `aster-frame` in the format of a Git URL.

However, if the new crate is within the Asterinas repository itself, the dependency on `aster-frame` should be specified using a relative path instead of a Git URL. I have added a FIXME for this issue and will address it soon.